docs/getting-started/v3-migration.md

```diff
@@ -53,36 +53,36 @@ Chart.js 3.0 introduces a number of breaking changes. Chart.js 2.0 was released
 
 ### Removed
 
+* `Chart.chart.chart`
+* `Chart.Controller`
+* `Chart.types`
+* `DatasetController.addElementAndReset`
+* `DatasetController.createMetaData`
+* `DatasetController.createMetaDataset`
+* `Element.getArea`
+* `Element.height`
+* `Element.initialize`
+* `Element.inLabelRange`
 * `helpers.addEvent`
 * `helpers.aliasPixel`
 * `helpers.configMerge`
 * `helpers.indexOf`
 * `helpers.lineTo`
-* `helpers.min`
 * `helpers.max`
+* `helpers.min`
 * `helpers.nextItem`
 * `helpers.numberOfLabelLines`
 * `helpers.previousItem`
 * `helpers.removeEvent`
 * `helpers.roundedRect`
 * `helpers.scaleMerge`
 * `helpers.where`
-* `Chart.Controller`
-* `Chart.chart.chart`
-* `Chart.types`
-* `DatasetController.addElementAndReset`
-* `DatasetController.createMetaData`
-* `DatasetController.createMetaDataset`
-* `Element.getArea`
-* `Element.height`
-* `Element.initialize`
-* `Element.inLabelRange`
 * `IPlugin.afterScaleUpdate`. Use `afterLayout` instead
 * `Line.calculatePointY`
 * `Scale.getRightValue`
+* `Scale.handleDirectionalChanges` is now private
 * `Scale.mergeTicksOptions`
 * `Scale.ticksAsNumbers`
-* `Scale.handleDirectionalChanges` is now private
 * `Scale.tickValues` is now private
 * The tooltip item's `x` and `y` attributes were removed. Use `datasetIndex` and `index` to get the element and any corresponding data from it
```

Comment:
Should this be at top to be consistent with other sections? Or should verbose lines in other sections rather be at bottom to be in sorted order?

Reply:
Ok. I moved the others to the bottom to keep them consistent